package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@octokit/plugin-throttling.js",
+  "version": "0.0.1",
+  "description": "Automatic rate limiting plugin for octokit",
+  "main": "lib/index.js",
+  "scripts": {
+    "pretest": "standard lib/",
+    "test": "mocha test/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/octokit/plugin-throttling.js.git"
+  },
+  "author": "Simon Grondin (http://github.com/SGrondin)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/octokit/plugin-throttling.js/issues"
+  },
+  "homepage": "https://github.com/octokit/plugin-throttling.js#readme",
+  "standard": {
+    "globals": [
+      "describe",
+      "before",
+      "beforeEach",
+      "afterEach",
+      "after",
+      "it",
+      "expect",
+      "cy"
```

Comment:
we can remove `cy` because we don’t have cypress tests at this point

Reply:
👍